docs/extend/extend-apiml/api-mediation-onboard-a-sprint-boot-rest-api-service.md

```diff
@@ -256,15 +259,16 @@ The first step to onboard a REST API with the Zowe ecosystem is to add enabler a
 
     * **mfaas.discovery.serviceId**
 
-         Specifies the service instance identifier to register in the API Layer installation. The service ID is used in the URL for routing to the API service through the gateway. The service ID uniquely identifies instances of a micro service in the API mediation layer. The system administrator at the customer site defines this parameter.  
+         Specifies the service instance identifier to register in the API ML installation. The service ID is used in the URL for routing to the API service through the gateway. The service ID uniquely identifies instances of a microservice in the API ML. The system administrator at the customer site defines this parameter.  
 
         **Important!**  Ensure that the service ID is set properly with the following considerations:
 
-         * When two API services use the same service ID, the API gateway considers the services to be clones. An incoming API request can be routed to either of them.
+         * When two API services use the same service ID, the API Gateway considers the services to be clones. An incoming API request can be routed to either of them.
          * The same service ID should be set for only multiple API service instances for API scalability.
          * The service ID value must contain only lowercase alphanumeric characters.
          * The service ID cannot contain more than 40 characters.
          * The service ID is linked to security resources. Changes to the service ID require an update of security resources.
+         * The service ID must match the `spring.application.name` parameter.
```

Comment:
Can we somehow highlight it? It's super important :-)

Reply:
This is already a bullet under "Important". If it deserves more attention we can put this point at the top of the list and add an explanation of what happens if the ID does not match the spring.application.name parameter. Can you edit this point to explain what happens if the service ID and spring.application.parameter don't match?